### ISSUE TITLE
django authenticator

### ISSUE THREAD

**Issue by figoyouwei** [figoyouwei/taipy_success#29]
Hey,

One of the purpose for tonight's meeting is for me to fully understand the taipy.auth mechanism in great details.

For example, I'd like to load my Django database which could be Postgres with user data and implement a Django authenticator.
https://github.com/figoyouwei/taipy_success/blob/main/auths/practice_authenticator.py

**So First, what's the key idea behind this?** 
I think the key is tp_enterprise.login() with a custom Authenticator, right?

**Second, could you implement one, already?** 
e.g. you set up a Django database in your local machine for testing.

**Comment by figoyouwei** [comment]
Yeah, please put this on top priority for this week to provide a solution codebase given your client is using Django with Postgres 13.

The basic idea is simple, Django provides by default a function called `check_password`. 
`from django.contrib.auth.hashers import check_password
`

What this function does is check the user-input password and return true or false, that's it. So you could provide a workaround solution using check_password, that would be the simplest. 

Then I could generalize the solution to work with any checking mechanism, literally just write another check_password function. So that's the idea:)

**Comment by figoyouwei** [comment]
A typical data would like like this below.
[users_data.csv](https://github.com/user-attachments/files/17536968/users_data.csv)

**Comment by AlexandreSajus** [comment]
Hey! I sent you an email with a package that should do this feature
